part-two/systems-design/README.md

```diff
@@ -1,33 +1,33 @@
 # Systems design exercise
 
-In this exercise we'll be designing a key component of Knock: our message delivery pipeline.
+In this exercise we'll be designing a key component of Knock: our **message delivery pipeline**.
 
-In Knock we take in customer API requests that "trigger" a defined workflow for one or more recipients, and turn them into notification messages that should be delivered to a provider using some predefined configuration.
+At Knock we accept customer API requests that "trigger" a defined workflow for one or more recipients, and turn them into notification messages that should be delivered to a provider using some predefined configuration.
 
-Our ingestion API and workflow execution service are already defined, and are **not** in scope for this exercise. Please imagine that our workflow service is ready to "produce" well-formed notification messages that are ready to be delivered by our delivery service. It's that service that we're going designing here, and how those messages are sent to your delivery service is up to you to define.
+Let's assume our ingestion API and workflow execution service are already defined, and are **not** in scope for this exercise. Imagine that our workflow service is ready to "produce" well-formed notification messages that are ready to be delivered by our *delivery service*. We'll design that *delivery service* here. How messages are sent to your delivery service is up to you.
 
 ## Service definition
 
-Our deliver service will integrate with many different providers to send out across different channels. We'll initially look to support emails, SMS, push messages, and chat providers like Slack, Discord and Teams. Our customers provide the appropriate configuraion for each provider, which we store in a "configuration service".
+Our deliver service will eventually integrate with many different providers to send out across different channels. Initially we'll want to support emails, SMS, push messages, and chat providers (Slack, Discord, and Teams). Our customers provide the appropriate configuration for each provider, which we store in a "configuration service". You can assume that the delivery service can send as many requests as it wants to the configuration service, which responds instantly at any scale.
 
 The delivery service should:
 
-- Process each notification at most once
-- Call the appropropriate delivery provider (sender) for the notification
-- Handle retries of messages, with exponential backoffs
-- Record delivery logs for all calls made to downstream services
-- Handle the result of the notification success or failure and pass this result to other services
+- Process each notification at most once.
+- Call the appropropriate delivery provider for the notification.
+- Handle retries of messages, with exponential backoffs.
+- Record an *audit trail* delivery logs for all calls made to downstream services. This trail should be eventually be available to customers.
```

Comment:
Let's leave this in but remove the out of scope requirement for the message logging service and then expand on what we mean by "available to customers"